lib/widgets/drop_down.dart

```diff
@@ -1,42 +1,52 @@
 import 'package:dropdown_button2/dropdown_button2.dart';
 import 'package:flutter/material.dart';
 
-class DropDown extends StatefulWidget {
-  List<String> items;
+class DropDown extends StatelessWidget {
+  final String? dropdownValue;
+  final List<String> items;
+  final Function didSelectItemHandler;
```

Comment:
This function seems to be missing the return type and which arguments it accepts. I guess you want this?
```suggestion
  final void Function(String) didSelectItemHandler;
```

Reply:
done